test/unit/Sync.spec.ts

```diff
@@ -0,0 +1,163 @@
+
+    const wrapper1 = sandbox("flick1");
+    wrapper1.style.width = "199px";
+    wrapper1.className = "flicking-viewport";
+    wrapper1.innerHTML = `
+      <div class="flicking-camera">
+        <div style="width: 200px; height: 200px;"><p></p></div>
+        <div style="width: 200px; height: 200px;"><p></p></div>
+        <div style="width: 200px; height: 200px;"><p></p></div>
+        <div style="width: 200px; height: 200px;"><p></p></div>
+        <div style="width: 200px; height: 200px;"><p></p></div>
+        <div style="width: 200px; height: 200px;"><p></p></div>
+      </div>
+    `;
+    flicking1 = new Flicking(wrapper1);
+  });
+
+  afterEach(() => {
+    cleanup();
+    flicking0.destroy();
+    flicking1.destroy();
+  });
+
+  it("main flicking should move with other flickings", async () => {
+    // Given
+    flicking0.addPlugins(new Sync({
+      type: "camera",
+      synchronizedFlickingOptions: [
+        {
+          flicking: flicking0
+        },
+        {
+          flicking: flicking1
+        }
+      ]
+    }));
+    await waitEvent(flicking0, "ready");
+
+    // When
+    void flicking0.control.moveToPosition(200, 0);
+
+    // Then
+    expect(flicking1.camera.position).not.to.equal(100);
+  });
+
+  it("other flickings should move main flicking", async () => {
+    // Given
+    flicking0.addPlugins(new Sync({
+      type: "camera",
+      synchronizedFlickingOptions: [
+        {
+          flicking: flicking0
+        },
+        {
+          flicking: flicking1
+        }
+      ]
+    }));
+    await waitEvent(flicking1, "ready");
+
+    // When
+    void flicking1.control.moveToPosition(200, 0);
+
+    // Then
+    expect(flicking0.camera.position).not.to.equal(100);
+  });
+
+  it("active panel should have active class", async () => {
+    // Given
+    flicking0.addPlugins(new Sync({
+      type: "index",
+      synchronizedFlickingOptions: [
+        {
+          flicking: flicking0,
+          isSlidable: true
+        },
+        {
+          flicking: flicking1,
+          activeClass: "flicking-thumbnail-active"
+        }
+      ]
+    }));
+    await waitEvent(flicking0, "ready");
+
+    // When
+    void flicking0.moveTo(1, 0);
+    tick(100);
+
+    // Then
+    expect(flicking1.getPanel(0).element.className.includes("flicking-thumbnail-active")).equal(false);
+    expect(flicking1.getPanel(3).element.className.includes("flicking-thumbnail-active")).equal(true);
+  });
+
+  it("slidable flicking should move other flickings", async () => {
+    // Given
+    flicking0.addPlugins(new Sync({
+      type: "index",
+      synchronizedFlickingOptions: [
+        {
+          flicking: flicking0,
+          isSlidable: true
+        },
+        {
+          flicking: flicking1
+        }
+      ]
+    }));
+    await waitEvent(flicking0, "ready");
+
+    // When
+    void flicking0.moveTo(1, 0);
+    tick(100);
+
+    // Then
+    expect(flicking1.camera.position).not.to.equal(100);
```

Comment:
How about checking the `index` instead of position?

Reply:
Added index check for tests with sync type 'index' .